### ISSUE TITLE
Login page-Login tab requires too many swipes with TalkBack

### ISSUE THREAD

**Issue by hydegeoff** [mmiron-HC/WST-RSAMS#25]
**Problem:**
When using TalkBack on the login screen, it requires too many swipes when navigating the tabs as described below:
Note: Place the cursor on the H1 Log in title and then start swiping: (the number represents the number of swipes)
1.	Primary tabs
2.	Login Active tab link
3.	Login 
4.	Active Tab 
5.	Reset Password 
6.	Username 

**Expected Result:**
There are too many swipes require to navigate the tabs on the page.  Talkback should only announce #2, #5, and #6.
Tested on Samsung S10 phone in a Chrome browser with Google TalkBack.
This happen in EN and FR page

Test URL: https://hcdev.openplus.ca/en/user/login

**Comment by hydegeoff** [comment]
Note I have verified that this works properly in the following WET example when using my phone, Chrome, and Talkback.
https://wet-boew.github.io/v4.0-ci/demos/tabs/tabs-en.html

**Comment by davidgotlieb** [comment]
Unable to reproduce as of Mid Jan, may have been fixed with other issues that were resolved.

**Comment by echouin** [comment]
From OP (09 Mar 22): 

- Attempted to use android emulators but I was unable to find one that had the TalkBack feature. I was able to briefly borrow a Samsung Galaxy, after placing cursor on the title and swiping it announced ﻿login active tab link, reset your password, username, and password.
- We were unable to duplicate the problem. As per our findings, it worked when he tried it on an android phone. It's possible﻿ it was fixed as part of an update, but we have no way of confirming that.

**Comment by hydegeoff** [comment]
Issue resolved in French and English

**Comment by hydegeoff** [comment]
This issue was accidentally closed.  This issue has been retested on Samsung S10e with Google talkback on and it still exists, there is another related with a video, if I can find the video add it to this ticket.

**Comment by echouin** [comment]
Could I please get an update on this? Has it been resolved on Prod?
